scIB/integration.py

```diff
@@ -283,6 +287,18 @@ def runBBKNN(adata, batch, hvg=None):
         corrected = bbknn.bbknn(adata, batch_key=batch, neighbors_within_batch=25, copy=True)
     return corrected
 
+
+def runSaucie(adata, batch):
```

Comment:
could you add in a docstring here what tutorial you got the parametrizations from?